ocs_ci/ocs/ui/page_objects/storage_system_details.py

```diff
@@ -10,9 +10,14 @@ def __init__(self):
 
     def nav_details_overview(self):
         logger.info("Click on Overview tab")
-        if (
+        if self.ocp_version_semantic >= version.VERSION_4_14:
+            self.do_click(
+                self.validation_loc["storagesystems_overview"], enable_screenshot=True
+            )
+
+        elif (
             self.ocp_version_semantic == version.VERSION_4_11
```

Comment:
could you please remove this block. It will not run against release-4.14 branch. Also comparing versions 
`self.ocp_version_semantic == version.VERSION_4_11 or self.ocs_version_semantic == version.VERSION_4_10`
seems not right


```
       elif (
            self.ocp_version_semantic == version.VERSION_4_11
            or self.ocs_version_semantic == version.VERSION_4_10
        ):
            self.do_click(
                self.validation_loc["overview_odf_4_10"], enable_screenshot=True
            )
```